bin/Powheg/production/2017/13TeV/GluGluToXToZZTo2B2Tau/JHUGen.input

```diff
@@ -0,0 +1,3 @@
+DecayMode1=2 DecayMode2=-289 ReweightDecay WidthSchemeIn=3 ReadPMZZ
```

Comment:
can you remind me what does "DecayMode2=-289" mean

Reply:
The "DecayMode2=-289" follows the rule described in the JHU manual [1] to require a specific decay mode.
In our case we want to produce Z->bb, so I'm applying the syntax -p1*p2 = -17*17 = -289 .

[1] https://spin.pha.jhu.edu/manJHUGenerator.v7.5.2.pdf